packages/wagmi/src/index.ts

```diff
@@ -1,12 +1,16 @@
-import { IBindings, RequiredSigner } from '@lens-protocol/react-web';
+import { IBindings } from '@lens-protocol/react-web';
 import { invariant } from '@lens-protocol/shared-kernel';
 import { providers } from 'ethers';
-import { SwitchChainNotSupportedError } from 'wagmi';
-import { fetchSigner, getNetwork, getProvider, switchNetwork } from 'wagmi/actions';
+import { PublicClient, SwitchChainNotSupportedError, WalletClient } from 'wagmi';
+import { getNetwork, getPublicClient, getWalletClient, switchNetwork } from 'wagmi/actions';
+import 'wagmi/window';
```

Comment:
what is this import for? won't it break in react-native or on server-side?

Reply:
good spot, this was for previous implementation when using `window.etherum` - it is now not needed 

i didn't get any feedback it wasn't used in my editor because it's not a var 🫣

will remove, thanks!